modules/update-cvo-conditions.adoc

```diff
@@ -0,0 +1,65 @@
+
+// When `RetrievedUpdates` is False, reason will be set to explain why, as discussed in the following subsections.
+// In all cases, the impact is that the cluster will not be able to retrieve recommended updates, so cluster admins will need to monitor for available updates on their own or risk falling behind on security or other bugfixes.
+// When CVO is unable to retrieve recommended updates the CannotRetrieveUpdates alert will fire containing the reason.
+// This alert will not fire when the reason updates cannot be retrieved is NoChannel.
+
+NoUpstream:: No upstream server has been set to retrieve updates.
++
+Set `spec.upstream` in `ClusterVersion` to point to a link:https://github.com/openshift/cincinnati/blob/master/docs/design/openshift.md[Cincinnati server] to receive updates.
+
+InvalidURI:: The configured upstream URI is not valid.
++
+Configure the upstream URI by setting `spec.upstream` in `ClusterVersion` to point to a valid link:https://github.com/openshift/cincinnati/blob/master/docs/design/openshift.md[Cincinnati URI].
+
+InvalidID:: The configured clusterID is not a valid UUID.
++
+To configure the clusterID, set `spec.clusterID` to a valid UUID. The UUID should be unique for each cluster. The default value is used report Telemetry and Insights. It may also be used by the CVO when making Cincinnati requests, so that Cincinnati can return update recommendations tailored to the specific cluster.
+
+NoArchitecture:: The set of architectures has not been configured. This is a result of a CVO coding error. There is no mitigation except to upgrade to a new release image with a fixed CVO.
+
+Impact:: The cluster is unable to retrieve recommended updates. This is a result of a CVO coding error. There is no mitigation except to upgrade to a new release image with a fixed CVO.
+
+NoCurrentVersion:: The cluster version does not have a semantic version assigned and is unable to calculate valid upgrades. This is the result of a release-image creation error. There is no mitigation except to upgrade to a new release image with a fixed CVO.
+
+NoChannel:: The update channel is not configured.
++
+To fix, set the channel to a valid value. For example, stable-4.8.
+
+InvalidCurrentVersion:: The cluster version does not have a semantic version assigned and is unable to calculate valid upgrades. This is the result of a release-image creation error. There is no mitigation except to upgrade to a new release image with a fixed CVO.
+
+InvalidRequest:: The CVO is unable to construct a valid Cincinnati request. This is a result of a CVO coding error. There is no mitigation except to upgrade to a new release image with a fixed CVO.
+
+RemoteFailed:: The CVO is unable to connect to the configured upstream. This may be caused by a misconfigured upstream URI. This may also be caused by networking or connectivity issue between the CVO and Cincinnati server. It may also be caused by a crashed or broken Cincinnati server.
```

Comment:
Cincinnati is the name of the API, so "a valid Cincinnati request" above makes sense to me.  But for "Cincinnati server", it may be better to use "Update Service" as seen [here][1].

[1]: https://docs.openshift.com/container-platform/4.8/updating/understanding-the-update-service.html